### PR TITLE
OpenSSL Tracing Demo

### PR DESCRIPTION
**Pull request by oazizi000**

### CODE REVIEW

**Review by htroisi — APPROVED** [review]

File: openssl-tracer/ssl_client_server/client.crt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```
Comment (aimichelle) [inline openssl-tracer/ssl_client_server/client.crt:1]: Should we actually be checking these certs in? Instead, should we just give them steps for how to generate their own certs? It'll be annoying to renew these certs when they expire later, for example.

Comment (oazizi000) [reply openssl-tracer/ssl_client_server/client.crt:1]: Yeah, I'm in the process of giving instructions instead.

**Review by zasgar — CHANGES_REQUESTED** [review]
Change certs/fix licenses as discussed.

File: openssl-tracer/Makefile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+openssl_prober: openssl_tracer.cc probe_deployment.cc
```
Comment (htroisi) [inline openssl-tracer/Makefile:17]: Should this be `openssl_tracer`?

**Review by aimichelle — APPROVED** [review]

**Pull request merged**